modules/Microsoft.Consumption/budgets/.test/default/deploy.test.bicep

```
@@ -0,0 +1,29 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. A short identifier for the kind of deployment .Should be kept short to not run into resource-name length-constraints')
+param serviceShort string = 'cbdef'
```

All `def `should probably be `com` now?
```suggestion
param serviceShort string = 'cbcom'
```

---

Updated via automation :)